server/src/main/resources/application.raft.example.yml

```diff
@@ -76,6 +72,15 @@ seata:
     # support: nacos 、 eureka 、 redis 、 zk  、 consul 、 etcd3 、 sofa
     type: file
     preferred-networks: 30.240.*
+    seata:
+      server-addr: 127.0.0.1:8081
+      cluster: default
+      namespace: public
+      heartbeat-period: 5000
+      metadata-max-age-ms: 30000
```

Comment:
If the type is not configured as seata here, how should the username and password for logging into the console be set?